### ISSUE TITLE
Activity vocab (FOAF v vCard Ontology)

### ISSUE THREAD

**Issue by riannella** [w3c/web-annotation#375]
I noticed the use of the as:generator property from the Activity Vocab CR.
They specify that https://www.w3.org/TR/activitystreams-vocabulary/#dfn-generator
describes an Object, which could be a Person (Actor_ - and they recommend the vCard Ontology for further properties: https://www.w3.org/TR/2016/CR-activitystreams-core-20160906/#actors

But the Annotation CR recommends FOAF?

**Comment by azaroth42** [comment]
Yes, we re-use some but not all of the Activity Streams classes and properties. We also, for example, don't use their notion of a Note, or a Link.  FOAF is more broadly adopted, and especially as AS hasn't even made it through CR yet, we have limited our use of it to those features we couldn't find in existing normative specifications.

Hope that helps?

**Comment by riannella** [comment]
That's not my point. If you reuse a property from another spec, then you "should" (RFC2119)  respect its recommendations about that property.

**Comment by azaroth42** [comment]
We do. The range of as:generator is Object U Link. We take the Object route. So the object of the triple is an as:Object (by inference from the predicate) and we also assert that it is a Person, Organization or Software.  For Software, that's an as:Application, and foaf:Person and foaf:Organization for the other two.  as:Application is a subClass of as:Object, so in that case we're doing exactly the same as Activity Streams.

**Comment by iherman** [comment]
Discussed on call 2016-11-11: it is too late change at this point, postponing.

See: http://www.w3.org/2016/11/11-annotation-irc#T16-11-31
